hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -677,16 +677,17 @@ cdef class PureMarketMakingStrategy(StrategyBase):
                                         buy.size, buy.price)
             quote_size = buy.size * buy.price * (Decimal(1) + buy_fees.percent)
             if quote_balance < quote_size_total + quote_size:
-                quote_size = quote_balance - quote_size_total
-                base_size = quote_size / buy.price * (Decimal(1) - buy_fees.percent)
-                buy.size = market.c_quantize_order_amount(self.trading_pair, base_size, buy.price)
+                self.logger().info(f"Insufficient balance: Quote order size {quote_size} is above available quote balance {quote_balance - quote_size_total}.")
```

Comment:
I think it'd be more informative for the message to be more like
`Insufficient balance: Buy order (price: XXX, size: xxx) is omitted, {self._quote_asset} available balance: XXX`

Reply:
Done.